api/src/integration/java/org/pmiops/workbench/TestWebMvcConfig.java

```diff
@@ -15,4 +19,23 @@
     public WorkbenchEnvironment workbenchEnvironment() {
         return new WorkbenchEnvironment(true, "appId");
     }
+
+    /**
+     * The loading of the spring application context for ApplicationTest
+     * was throwing a BeanCreationException: Scope 'request' is not active
+     * for the current thread. Since WorkbenchConfig is request scoped using
```

Comment:
FirecloudConfig does this - I guess the difference is that it injects it into the provider rather than autowiring into the configuration class itself.